_docs/tasks/telemetry/querying-metrics.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Querying Istio Metrics
+
+overview: This task shows you how to query for Istio Metrics using Prometheus.
+
+order: 15
+
+layout: docs
+type: markdown
+---
+{% include home.html %}
+
+This task shows you how to query for Istio Metrics using Prometheus. As part of
+this task, you will install the Prometheus Istio addon and use the web-based
+interface for querying metric values.
+
+The [BookInfo]({{home}}/docs/guides/bookinfo.html) sample application is used as
+the example application throughout this task.
+
+## Before you begin
+* [Install Istio]({{home}}/docs/setup/) in your cluster and deploy an
+  application.
+
+## Querying Istio Metrics
+
+1. To query the metrics provided by Mixer, first install the Prometheus add-on.
+
+    In Kubernetes environments, execute the following command:
+
+    ```bash
+    kubectl apply -f install/kubernetes/addons/prometheus.yaml
+    ```
+
+1. Verify that the service is running in your cluster.
+
+    In Kubernetes environments, execute the following command:
+
+    ```bash
+    kubectl -n istio-system get svc prometheus
+    ```
+
+    The output will be similar to:
+
+    ```
+    NAME         CLUSTER-IP     EXTERNAL-IP   PORT(S)    AGE
+    prometheus   10.59.241.54   <none>        9090/TCP   2m
+    ```
+
+1. Send traffic to the mesh.
+
+    For the BookInfo sample, visit `http://$GATEWAY_URL/productpage` in your web
+    browser or issue the following command:
+
+   ```bash
+   curl http://$GATEWAY_URL/productpage
+   ```
+
+   Note: `$GATEWAY_URL` is the value set in the
+   [BookInfo]({{home}}/docs/guides/bookinfo.html) guide.
+
+1. Open the Prometheus UI.
+
+    In Kubernetes environments, execute the following command:
+
+    ```bash
+    kubectl -n istio-system port-forward $(kubectl -n istio-system get pod -l app=prometheus -o jsonpath='{.items[0].metadata.name}') 9090:9090 &   
+    ```
```

Comment:
Just curious.. do you think it would show better if this is wired up with istio-ingress?  I think it is easier for user to see it that way and also not worry about killing the kubectl process afterwards.